server/world/sound/block.go

```diff
@@ -81,8 +81,8 @@ type Note struct {
 	Pitch int
 }
 
-// ItemFrameAdd is a sound played when an item is added to an item frame.
-type ItemFrameAdd struct{ sound }
+// AddItem is a sound played when an item is added to an item frame or campfire.
```

Comment:
Should be renamed to `ItemAdd` to match naming of other sounds, i.e `BlockPlace`